cypress/integration/recieve-notice/recieve-notice-after-checkout.spec.js

```diff
@@ -0,0 +1,196 @@
+      waiter: NewPatronNoticeTemplate.waitLoading,
+    });
+
+    NewPatronNoticeTemplate.createTemplate(testPatronNoticeTemplate);
+    NewPatronNoticeTemplate.checkTemplate(testPatronNoticeTemplate);
+    cy.visit(settingsMenu.circulationPatronNoticePoliciesPath);
+    newPatronNoticePolicies.createPolicy(testPatronNotice);
+    newPatronNoticePolicies.savePolicy();
+    testPatronNotice.templateId = testPatronNoticeTemplate.name;
+    testPatronNotice.format = 'Email';
+    testPatronNotice.action = 'Check out';
+    newPatronNoticePolicies.addNotice(testPatronNotice);
+    newPatronNoticePolicies.savePolicy();
+    newPatronNoticePolicies.checkPolicy(testPatronNotice.name);
+  });
+
+  it('C347621 Check that user can receive notice with multiple items after finishing the session "Check out" by clicking the End Session button', () => {
+    cy.visit(settingsMenu.circulationRulesPath);
+    circulationRules.clearCirculationRules();
+    circulationRules.fillInPriority();
+    circulationRules.fillInFallbackPolicy({
+      loanPolicyName: defaultLoanPolicy.name,
+      overdueFinePolicyName: defaultOverdueFinePolicy.name,
+      lostItemFeePolicyName: defaultLostItemFeePolicy.name,
+      requestPolicyName: defaultRequestPolicy.name,
+      noticePolicyName: testPatronNotice.name,
+      materialTypeName: defaultMaterialType.name,
+    });
+    circulationRules.fillInPolicy({
+      priorityType: 'g ',
+      priorityTypeName: testPatronGroup.group,
+      loanPolicyName: defaultLoanPolicy.name,
+      overdueFinePolicyName: defaultOverdueFinePolicy.name,
+      lostItemFeePolicyName: defaultLostItemFeePolicy.name,
+      requestPolicyName: defaultRequestPolicy.name,
+      noticePolicyName: testPatronNotice.name,
+    });
+    circulationRules.saveCirculationRules();
+    cy.visit(TopMenu.checkOutPath);
+    checkoutActions.checkOutItem(USER_BARCODE, ITEM_BARCODE);
+    cy.visit(TopMenu.circulationLogPath);
+    SearchPane.searchByUserBarcode(USER_BARCODE);
+    SearchPane.verifyResultCells();
+  });
+
+  after(() => {
+    cy.visit(settingsMenu.circulationRulesPath);
+    circulationRules.clearCirculationRules();
+    circulationRules.fillInPriority();
+    circulationRules.fillInFallbackPolicy({
+      loanPolicyName: 'example-loan-policy',
+      overdueFinePolicyName: 'overdue-fine-policy',
+      lostItemFeePolicyName: 'lost-item-fee-policy ',
+      requestPolicyName: 'allow-all',
+      noticePolicyName: 'send-no-notices',
+    });
+    circulationRules.fillInPolicy({
+      priorityType: 'm ',
+      priorityTypeName: 'book',
+      loanPolicyName: 'one-hour',
+      overdueFinePolicyName: 'overdue-fine-policy',
+      lostItemFeePolicyName: 'lost-item-fee-policy',
+      requestPolicyName: 'allow-all',
+      noticePolicyName: 'send-no-notices',
+    });
+    circulationRules.saveCirculationRules();
+    cy.getUsers({ query: `personal.lastName='${userLastName}'` }).then(() => {
```

Comment:
remember userId in before and run remove directly by userId instead of extra query by userName